File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L4/01/L4.ts
import * as readline from 'readline';

function hitLuas(jari2: number): number {
    return jari2 * jari2 * 3.14159;
}

function total(luas: number, panjang: number): number {
    return luas * panjang;
}

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter the radius and length of a cylinder: ", (answer) => {
    const [jari2, panjang] = answer.split(' ').map(Number);
    const luas = hitLuas(jari2);
    const totalVolume = total(luas, panjang);

    console.log(`The area is ${luas}, The volume of the cylinder is ${totalVolume}`);
    rl.close();
});

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-02/plagiarized/L4/01/L4.kt
package IR

import java.util.*

object L4 {
    fun hitLuas(jari2: Double): Double {
        return jari2 * jari2 * 3.14159
    }

    fun total(luas: Double, panjang: Double): Double {
        return luas * panjang
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val input = Scanner(System.`in`)
        var jari2 = 0.0
        var panjang = 0.0
        print("Enter the radius and length of a cylinder: ")
        jari2 = input.nextDouble()
        panjang = input.nextDouble()
        val luas = hitLuas(jari2)
        val total = total(luas, panjang)

        print("The area is $luas , ")
        println("The volume of the cylinder is $total")
    }
}
